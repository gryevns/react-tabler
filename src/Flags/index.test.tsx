import React from "react";
import { shallow } from "enzyme";

import Flag from "./index";

describe("Flag", () => {
    it("should match country snapshot", () => {
        const wrapper = shallow(<Flag country="gb" />);
        expect(wrapper).toMatchSnapshot();
    });
    it("should match size snapshot", () => {
        ["xl" as const, "lg" as const, "md" as const].map(size => {
            const wrapper = shallow(<Flag country="gb" size={size} />);
            expect(wrapper).toMatchSnapshot();
        });
    });
});
