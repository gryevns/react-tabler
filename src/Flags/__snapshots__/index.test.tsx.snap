// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flag should match country snapshot 1`] = `
<span
  className="flag flag-country-gb"
/>
`;

exports[`Flag should match size snapshot 1`] = `
<span
  className="flag flag-country-gb"
/>
`;

exports[`Flag should match size snapshot 2`] = `
<span
  className="flag flag-country-gb"
/>
`;

exports[`Flag should match size snapshot 3`] = `
<span
  className="flag flag-country-gb"
/>
`;

exports[`Flag should match size snapshot 4`] = `
<span
  className="flag flag-country-gb"
/>
`;
